### PR TITLE
Fix 'BadPin' issue

### PR DESCRIPTION
**Pull request by tatiNo5**

### CODE REVIEW

File: MiraclIdentityVerificationApp/Controllers/HomeController.cs [hunk 1]
```diff
@@ -116,7 +116,7 @@ public async Task<ActionResult> Login(string identity)
             return Redirect(authorizationUri);
         }
 
-        public async Task<ActionResult> Logout(string data)
```
Comment (VMitov) [inline MiraclIdentityVerificationApp/Controllers/HomeController.cs]: Is this related to the issue?

Comment (tatiNo5) [reply MiraclIdentityVerificationApp/Controllers/HomeController.cs]: No, there was a warning for this and as it's a small change I've added it here too.

Comment (VMitov) [reply MiraclIdentityVerificationApp/Controllers/HomeController.cs]: Can you put it in a separate commit?
